packages/p2p/src/components/my-ads/buy-ad-payment-methods-list.jsx

```diff
@@ -0,0 +1,202 @@
+                                            }}
+                                            leading_icon={
+                                                <Icon
+                                                    icon={
+                                                        payment_method_icon === 'BankTransfer' ||
+                                                        payment_method_icon === 'Other'
+                                                            ? `IcCashier${payment_method_icon}`
+                                                            : 'IcCashierEwallet'
+                                                    }
+                                                />
+                                            }
+                                            trailing_icon={
+                                                <Icon
+                                                    icon='IcDelete'
+                                                    onClick={() => onClickDeletePaymentMethodItem(payment_method)}
+                                                />
+                                            }
+                                            type='text'
+                                        />
+                                    )}
+                                </Field>
+                            )}
+                        </Formik>
+                    );
+                })}
+                {my_ads_store.payment_method_names.length < 3 && payment_methods_list.length > 0 && (
+                    <Formik enableReinitialize initialValues={{ payment_method: '' }}>
+                        {({ setFieldValue }) => (
+                            <Field name='payment_method'>
+                                {({ field }) => (
+                                    <div className='p2p-my-ads--border'>
+                                        <Autocomplete
+                                            {...field}
+                                            className='quick-add-modal--input'
+                                            is_alignment_top
+                                            autoComplete='off' // prevent chrome autocomplete
+                                            data-lpignore='true'
+                                            list_items={payment_methods_list}
+                                            onItemSelection={({ value }) => onClickPaymentMethodItem(value)}
+                                            onBlur={e => {
+                                                e.preventDefault();
+                                                setFieldValue('payment_method', '');
+                                            }}
+                                            required
+                                            leading_icon={<Icon icon='IcAddOutline' size={14} />}
+                                            trailing_icon={<></>}
```

Comment:
```suggestion
                                            trailing_icon={<></>}
```
Is adding an empty component here a must? 👀  Same question for the others

Reply:
Nopee, I have removed it 😅